ztools/x509/json.go

```diff
@@ -33,31 +35,31 @@ type auxKeyUsage struct {
 func (k KeyUsage) MarshalJSON() ([]byte, error) {
 	var enc auxKeyUsage
 	enc.Value = uint32(k)
-	if k & KeyUsageDigitalSignature > 0 {
+	if k&KeyUsageDigitalSignature > 0 {
```

Comment:
Oh true. That's weird, looks like my editor auto-formats things differently from `go fmt`. I'll fix my setup